test/clj/rems/db/test_user_settings.clj

```diff
@@ -0,0 +1,24 @@
+(ns ^:integration rems.db.test-user-settings
+  (:require [clojure.test :refer :all]
+            [rems.db.testing :refer [rollback-db-fixture test-db-fixture]]
+            [rems.db.user-settings :as user-settings]
+            [rems.db.users :as users]))
+
+(use-fixtures :once test-db-fixture)
+(use-fixtures :each rollback-db-fixture)
+
+(deftest test-user-settings
+  (users/add-user! "user1" {:eppn "user1"})
```

Comment:
These tests would not notice it if the SQL queries were missing WHERE conditions. For that you need a second user and then check that its settings are not changed when some other user's settings are changed.

Reply:
Done.